pom.xml

```diff
@@ -68,7 +68,7 @@
 
         <!-- Test dependencies -->
         <strimzi.version>0.46.0</strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
-        <kubernetes-client.version>6.13.5</kubernetes-client.version>
+        <kubernetes-client.version>7.1.0</kubernetes-client.version>
```

Comment:
Given https://github.com/fabric8io/kubernetes-client/issues/7037 we probably need to go to 7.2.0